includes/header.php

```diff
@@ -45,6 +45,7 @@
   <meta name="description" content="<?php echo $page_meta; ?>">
   <meta name="author" content="Phil Ewels">
   <meta name="color-scheme" content="light dark">
+  <meta name="theme-color" content="#206e43">
```

Comment:
```suggestion
  <meta name="theme-color" content="#1d9655">
```

How about the nf-core-color we use in the dark theme instead? 
<img width="1857" alt="Screenshot 2022-04-05 at 09 55 42" src="https://user-images.githubusercontent.com/6169021/161706435-056bb745-b2c9-4b10-a86f-a7d6f0e8f4b3.png">

Reply:
Yeah, if that works! I was just doing trial and error trying to find something that safari would load.